oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -64,7 +68,7 @@
  * <p>These credentials use the IAM API to sign data. See {@link #sign(byte[])} for more details.
  * </p>
  */
-public class ComputeEngineCredentials extends GoogleCredentials implements ServiceAccountSigner {
+public class ComputeEngineCredentials extends GoogleCredentials implements ServiceAccountSigner, IdTokenProvider {
```

Comment:
Is this going to require a major version bump? That is, will it break existing subclasses that do not implement IdTokenProvider?

Reply:
@chingor13  

thye'll just inherit all these changes

Reply:
This shouldn't be a breaking change.